ddl/foreign_key.go

```diff
@@ -549,3 +549,44 @@ func (h *foreignKeyHelper) getTableFromStorage(is infoschema.InfoSchema, t *meta
 	h.loaded[k] = result
 	return result, nil
 }
+
+func checkDatabaseHasForeignKeyReferred(is infoschema.InfoSchema, schema model.CIStr, fkCheck bool) error {
```

Comment:
I suggest that don't put too much test case into the same foldier. it will cause performance problem when to run with bazel.
It should be like ```ddl/metadatalocktest```.

Reply:
I will move this test into `ddl/foreignkeytest` in next PR